cookbooks/travis_packer_templates/recipes/default.rb

```diff
@@ -37,6 +37,17 @@
   )
 end
 
+file '/.node-attributes.yml' do
+  content YAML.dump(
+    JSON.parse(JSON.dump(node.attributes.to_h)).merge(
```

Comment:
`node.attributes` get all key: value pairs inside nodes - looking up based on precedence. --> and turning into hash `.to.h`  parsed to JSON first then to YML dump.